doc/sphinx/tutorial2.md

```diff
@@ -0,0 +1,81 @@
+---
+discourse: 4911,5164,5603,6756,8037
+---
+
+# Tutorial 2
+This tutorial will guide you through using Mir demos to see what Mir is capable
+of, and the usage of Mir libraries to build your own Wayland compositor.
+
+## Mir Demos
+Mir comes with a set of demos in its [example
+directory](https://github.com/canonical/mir/tree/main/examples) to showcase its
+abilities. This section will cover the installation and usage of these demos.
+
+
+### Installation
+Mir demos are available on Ubuntu as prebuilt binaries. For distros that don't
+have prebuilt binaries, examples can be built from source.
+
+<details>
+<summary> Installing Mir Demos on Ubuntu </summary>
+
+You can run the following command to install Mir demos and the drivers they
+use.
+```sh
+sudo apt install mir-demos mir-graphics-drivers-desktop
+```
+</details>
+
+<details>
+<summary>Building Mir Demos on Other Distros</summary>
+
+For Linux distributions that don't currently package Mir demos you need to
+build them yourself. (See [Getting Involved in
+Mir](how-to/getting_involved_in_mir)).
+</details>
+
+### Running
```

Comment:
This currently only describes running nested (via X11) on a desktop.  "using Mir demos to see what Mir is capable of" should also cover:

* running "natively" (by running `miral-app` from a VT or starting from the greeter)
* using an onscreen keyboard
* changing the decoration strategy
* mixing Wayland and X11 clients
* Remote desktop
* Onscreen keyboard
* nesting on Wayland host
* ...

Reply:
Oh boy, that's _a lot_. Perhaps a little too much for a tutorial? I can't really estimate how big each of those will be but I think it might be overwhelming to new users? 

Perhaps splitting each to a separate "Demo How-To" would be a little less overwhelming. What do you think?

Reply:
It really isn't that much

Reply:
Note that after asking @Saviq about nesting, he deemed it too advanced for the demo tutorial, and so it was removed.